docs/aws/README.md

```diff
@@ -15,38 +44,34 @@ KARPENTER_VERSION=$(curl -fsSL \
   | jq -r '.tag_name')
 ```
 
-### Create a Cluster
+## Create a Cluster
 
-Karpenter can run anywhere, including on self-managed node groups, [managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html), or [AWS Fargate](https://aws.amazon.com/fargate/).
-This demo will run Karpenter on Fargate, which means all EC2 instances added to this cluster will be controlled by Karpenter.
+Create a cluster with `eksctl`. The below configuration file specifies a basic
+cluster (name, region), an IAM role for Karpenter to use, and two fargate
+profiles. 
 
-```bash
-curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/"${KARPENTER_VERSION}"/docs/aws/eks-config.yaml \
-  | envsubst \
-  | eksctl create cluster -f -
-```
+The two fargate profiles host `kube-system` and the Karpenter service itself.
+This permits Karpenter to manage all EC2 instances. 
 
-Tag the cluster subnets with the required tags for Karpenter auto discovery.
+Karpenter itself can run anywhere, including on self-managed node groups,
+[managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html), 
+or [AWS Fargate](https://aws.amazon.com/fargate/).
 
-> If you are using a cluster with version 1.18 or below you can skip this step.
-More [detailed here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
+Karpenter will provision traditional instances on EC2. 
```

Comment:
I'd be ultra crisp that these instances run in your account. "Karpenter provisions EC2 instances in your account"